### PR TITLE
Update markdown documentation and close #23

### LINKED ISSUE #23: General documentation/markdown update
Preparing the way for subsequent PRs related to specific $\mathbf{(A \rightarrow B)}$ process steps, their scripts, the resulting stage $B$ and associated data files; first the markdown files of the repo are to be updated for consistency with updated code.

### PR DESCRIPTION
**Pull request by tlinjordet**
This adjusts the whole-repo documentation to be up-to-date for subsequent PRs on specific process steps. 
This PR closes #23 .

### CODE REVIEW

**Comment by tlinjordet** [comment]
This PR closes #23 .

File: datasets/08a_uncond-excluded/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+We observed geometric distorsions in 63% and 14% of the $[\mathrm{Ir} \mathrm{L}_4]^+$ and $[\mathrm{Ir} \mathrm{L}_2]^+$ outliers, respectively. These do not include the TMCs with ligands missing H atoms, which are 11% in both series. These features appear in the comment line of each TMC entry in the monodentate and bidentate `.xyz` files in this folder.
+
+The identifiers of the DFT-labeled TMCs shown in the figure above (Figure S8 of the SI) are as follows:
+
+$[\mathrm{Ir} \mathrm{L}_4]^+$ top row, left to right, identifed simply as `{Label ID}`: 3886, 4994, 6876, and 3490.
+
+$[\mathrm{Ir} \mathrm{L}_2]^+$ bottom row, left to right, identified as `{Ligand ID}-{Isomer}`: uncond_bi-min10k-1518-trans, uncond_bi-min10k-1812-trans, uncond_bi-min10k-8929-trans, uncond_bi-min10k-7426-trans.
+
+Both sets of outliers were extracted with the isolation forest method with contaminations parameters 0.010 ($[\mathrm{Ir} \mathrm{L}_4]^+$) and 0.015 ($[\mathrm{Ir} \mathrm{L}_2]^+$).
```
Comment (hkneiding) [inline datasets/08a_uncond-excluded/README.md:15]: Should be "contamination~~s~~ parameters"

File: DETAILS.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
 
-Every data workflow stage under [datasets/DETAILS.md](datasets/DETAILS.md) that relies on the code in the present repository can be reproduced by one Python script (`tmcinvdes/.../{process_verb}.py`) being called by two Bash scripts (`bash/{process_verb}-{denticity}-{specifications}.sh`). *(The `-{specifications}` can be used to disinguish Bash files where the Python script is used to produce more than one stage, i.e., for assembling TMC structures, and for parsing ORCA output files to label TMCs/ligands after ORCA calculations.)*
+Every data workflow stage under [datasets/DETAILS.md](datasets/DETAILS.md) that relies on the code in the present repository can be reproduced by one Python script (`tmcinvdes/.../{process_verb}.py`) being called by two Bash scripts (`bash/{process_verb}-{denticity}-{specifications}.sh`). Reproduction is not always perfect, and further details on this aspect are available [here](REPRODUCIBILITY.md). *(The `-{specifications}` can be used to disinguish Bash files where the Python script is used to produce more than one stage, i.e., for assembling TMC structures, and for parsing ORCA output files to label TMCs/ligands after ORCA calculations.)*
```
Comment (hkneiding) [inline DETAILS.md:5]: Should be "... can be used to dis**t**inguish Bash files ..."

**Review by hkneiding — APPROVED** [review]
Two typos to be fixed in following commits.

**Comment by tlinjordet** [comment]
Thanks!

**Pull request merged**